man/ocitools-generate.1.md

```diff
@@ -137,18 +137,19 @@ read the configuration from `config.json`.
   The special *PATH* `host` removes any existing PID namespace from
   the configuration.
 
-**--poststart**=CMD
-  Path to command to run in poststart hooks. This command will be run before
-  the container process gets launched but after the container environment and
-  main process has been created.
+**--poststart**=[CMD[:ARG]]
```

Comment:
On Tue, Aug 16, 2016 at 08:07:50PM -0700, Ma Shimiao wrote:
> 08/17/2016 11:01 AM, W. Trevor King:
> > Why the brackets around the whole thing? I'd expect:
> > 
> > *--poststart*=CMD[:ARG...]
> 
> brackets around the whole thing means it accepts more than one, it's
> a slice.
>
> > *--poststart*=CMD[:ARG...]
>
> looks like only ARG accepted as slice.

I'm used to brackets in man pages meaning “the enclosed bit is
optional” with “accepts more than one” being represented with an
ellipsis.  For example:

  $ man git-commit | head -n12 | tail -n6
       git commit [-a | --interactive | --patch] [-s] [-v] [-u<mode>] [--amend]
                  [--dry-run] [(-c | -C | --fixup | --squash) <commit>]
                  [-F <file> | -m <msg>] [--reset-author] [--allow-empty]
                  [--allow-empty-message] [--no-verify] [-e] [--author=<author>]
                  [--date=<date>] [--cleanup=<mode>] [--[no-]status]
                  [-i | -o] [-S[<keyid>]] [--] [<file>...]

Showing that <keyid> is an optional argument for -S and <file> can be
repeated.